lib/PTML/Doxygen.cpp

```diff
@@ -461,9 +478,13 @@ gatherReturnValueComments(const model::Binary &Binary,
                            model::Register::getRegisterName(Register).str());
         auto &Tag = Line->emplace_back(DoxygenToken::Types::Untagged,
                                        ": " + ReturnValue.Comment());
-        Tag.ExtraAttributes.emplace_back(ptml::attributes::ModelEditPath,
-                                         model::editPath::comment(*FT,
-                                                                  ReturnValue));
+        Tag.ExtraAttributes
+          .emplace_back(ptml::attributes::ActionContextLocation,
+                        serializedLocation(ranks::ReturnRegister,
+                                           FT->key(),
+                                           ReturnValue.key()));
```

Comment:
Can you try to make a final commit that introduces a `using namespace ptml` in this `.cpp` file and maybe also introduces a `toString` local function that aliases `serializedLocation`.

If it improves the readability of the file significantly, put it in the PR.

Reply:
I've done the changes you asked, they didn't help much.
I can push the commit/give you the diff if you want to take a look, but it all amounted to ~20 lines __barely__ changing.

Reply:
I've done the changes you asked, they didn't help much.
I can push the commit/give you the diff if you want to take a look, but it all amounted to ~20 lines _barely_ changing.